myutils.py

```diff
@@ -56,3 +60,20 @@ def get_latest_build(self):
         group_json = requests.get(
             self.OPENQA_URL_BASE + 'group_overview/262.json', verify=False).json()
         return group_json['build_results'][0]['build']
+
+    def shell_exec(self, cmd, log=False):
+        try:
+            if log:
+                self.logger.info(cmd)
+            output = subprocess.check_output(cmd, shell=True)
+            try:
+                o_json = json.loads(output)
```

Comment:
I think this is a bit dirty. Guessing that the output of some shell command is always JSON. I would prefere to make enable this path with an parameter or just skip it from here, and let the caller know if he want's json or not.

Reply:
yeah I somehow knew that you will comment here :D . ok :)